cores/piduino/WString.h

```diff
@@ -123,5 +123,8 @@ class StringSumHelper : public String
   StringSumHelper(double num) : String(num) {}
 };
 
+typedef char __FlashStringHelper;
+#define PROGMEM
+
```

Comment:
many libraries out there include <avr/pgmspace> and while we do not need all of the things defined there, we need at least the most commonly used defines. Having the file in avr subfolder will make those libraries work out of the box and that is I think what we need more, rather than teaching the users of the new pgmspace.h location (like we had to do with the ESP). I can hide all that in a separate folder that will be included in the build and not mess up the folder structure we have.